components/crm/.env.example

```
@@ -164,6 +164,34 @@ MULTI_TENANT_ENABLED=false
+# =============================================================================
+# Registers this instance in Consul and resolves downstream hosts (e.g.
+# plugin-auth) via Consul instead of static addresses. Default is OFF: when
+# SD_ENABLED=false a no-op Manager is wired, no Consul connection is attempted,
+# and Resolve returns the static fallback host unchanged.
+#
+# When SD_ENABLED=true, SD_ADVERTISE_ADDRESS is REQUIRED. Enabling discovery
+# with an empty advertise address aborts boot (ErrEmptyAdvertiseAddr).
+# Canonical SD_* names below are read by libsd.ConfigFromEnv().
+SD_ENABLED=false
+# Consul HTTPS API (host:port); central agents typically listen on :8501.
+SD_ADDRESS=localhost:8500
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SD_ADDRESS key should go before the SD_ENABLED key

(UnorderedKey)